digiroad2-oracle/src/main/resources/db/migration/V0_32__calibration_point_type_update.sql

```diff
@@ -0,0 +1,35 @@
+UPDATE CALIBRATION_POINT cpu SET "TYPE" = (SELECT CASE
+-- leave expired calibration points to original value 2
+		WHEN (cp.VALID_TO IS NOT NULL) THEN 2
```

Comment:
I would remove this and just let the expired ones with the type they have - doesn't matter which is.
And add an `where` clause to the end.

Reply:
fixed